modules/formulize/templates/admin/element_optionlist.html

```diff
@@ -54,5 +63,65 @@
     $("[name=cleardef]").blur();
   });
   <{/if}>
+  
+  <{if $content.type eq "select"}>
+  checkNAMES();
+  /**
+   *this function will consantly check for {USERNAMES} and {FULLNAMES}
+   */
+  function checkNAMES(){
+    var x=0,y=-1,z=-1;    //y for {FULLNAMES}  z for {USERNAMES}
+    for(x=0;x<length;x++){

```

Comment:
OK, I'll try to alert user when they enter {USERNAMES} or {FULLNAMES}.

The msg will be something like "if you want to keep one of these, everything else will be deleted. and the very first box will be come one of these. Do you want to apply this change?"  If user click yes, it will apply the change stated above, else it will delete either {U..} or {F..} in the box. 

Does this sounds good to you ?